lib/vmdb/console_methods.rb

```diff
@@ -28,6 +28,7 @@ def simulate_queue_worker(break_on_complete = false)
         else
           break_on_complete ? break : sleep(1.second)
```

Comment:
Possibly change this line to the following then (to avoid developer confusion):

```
log "No messages in the queue."
if break_on_complete
  log "Exiting" and break
else
  log "Waiting for more messages..." and sleep(1)
end
```